src/dtgtk/thumbnail.c

```diff
@@ -215,8 +215,6 @@ static void _image_get_infos(dt_thumbnail_t *thumb)
 {
   if(!dt_is_valid_imgid(thumb->imgid))
     return;
-  if(thumb->over == DT_THUMBNAIL_OVERLAYS_NONE)
-    return;
 
```

Comment:
That means we'll *always* read the metadata, even if not really needed. That may slow up a little thumb loading...
The ideal changes here would be to have a specific path to retrieve only reject values (because afaik it's the only data we show on thumbs without overlays)